### PR TITLE
Fix session validation bug

### PR DESCRIPTION
**Pull request by Suvink**
$subject

### CODE REVIEW

File: lib/src/models/session-data.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
```
Comment (thivi) [inline lib/src/models/session-data.ts]:
```suggestion
* Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
```

File: samples/config.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const config = {
-    "clientID": "",
-    "serverOrigin": "",
+    "clientID": "Mm2adXV2jnL9pfo2kCHIEi_ej3Aa",
+    "serverOrigin": "https://api.asgardeo.io/t/iconicto",
     "signInRedirectURL": "http://localhost:5000/login",
```
Comment (thivi) [inline samples/config.js]: Let's leave the values empty

File: samples/index.js [hunk 3]
```diff
@@ -63,6 +63,7 @@ app.get("/id", (req, res) => {
         //You may redirect the user to login endpoint here.
         // res.status(301).redirect("/login");
     } else {
+        console.log(req.cookies.ASGARDEO_SESSION_ID)
```
Comment (thivi) [inline samples/index.js]: We don't need it, do we?

File: samples/package.json [hunk 4]
```diff
@@ -15,7 +15,7 @@
     "nodemon": "^2.0.15"
   },
   "dependencies": {
-    "@asgardeo/auth-node-sdk": "^0.0.1",
+    "@asgardeo/auth-node-sdk": "file:../lib",
```
Comment (thivi) [inline samples/package.json]: Shall we use a version number here?

File: samples/index.js (hunk 3 above)
Comment (Suvink) [reply samples/index.js]: Just for debugging

**Review by thivi — APPROVED** [review]

**Pull request merged**